servers/eclipsia/metadata.json

```diff
@@ -0,0 +1,45 @@
+{
+    "id": "eclipsia",
+    "name": "Eclipsia",
+    "description": "Eclipsia is a mini-game server",
+    "addresses": [
+        "eclipsia.fr"
+    ],
+    "primaryAddress": "play.eclipsia.fr",
+    "minecraftVersions": [
+        "1.8.*",
+        "1.9.*",
```

Comment:
only pur versions that lunar client supports

Reply:
Done